backend/apps/main/models.py

```diff
@@ -15,6 +15,8 @@ class Chat(BaseModel):
     )
     title = models.CharField(max_length=500, null=True, blank=True)
     type = models.CharField(max_length=20, choices=TYPES)
+    from_user = models.ForeignKey(User, on_delete=models.CASCADE, related_name='chat_from_user')
+    to_user = models.ForeignKey(User, on_delete=models.CASCADE, related_name='chat_to_user')
 
```

Comment:
i think, we should add group field here. this field will help us to get chat title, if chat type is group.

Reply:
In first you create Group model in this file(main/models.py). Then add the field like group = models.ForeignKey('main.Group', models.CASCADE, null=True, blank=True)